### DISCUSSION TITLE
Does Livewire support route throttling / rate limiting?

### DISCUSSION THREAD

**Discussion by EricBusch** [livewire/livewire#1913] [4. General]
Hi all,

I'm trying to throttle a Livewire route. I've tried this:

```php
Route::middleware(['throttle:6,1'])->group(function () {
    Route::get('/', Scratchpad::class);
});
```

and this:

```php
Route::get('/', Scratchpad::class)->middleware(['throttle:6,1']);
```

and this (**_RouteServiceProvider_**)

```php
RateLimiter::for('web', function (Request $request) {
    return Limit::perMinute(3);
});
```

Neither seem to work. Am I doing something wrong or is this not currently supported?

Laravel: `8.11.2`
Livewire: `2.3.0`

Thanks
Eric

**Comment by chaudigv** [comment]
Livewire do support a **middleware_group**, check your `livewire/config.php` file
e.g.
`'middleware_group'  => ['web', 'throttle:6,1']`

**Reply by EricBusch** [reply]
Awesome, thanks!
